bitsandbytes/backends/xpu/ops.py

```diff
@@ -1,58 +1,214 @@
 from collections.abc import Sequence
+import ctypes as ct
 import warnings
 
 import torch
 
+from bitsandbytes.functional import _get_tensor_stream, get_ptr
+
 from ..._ops import register_kernel
-from ..utils import ipex_xpu, triton_available
+from ...cextension import ErrorHandlerMockBNBNativeLibrary, lib
+from ..utils import triton_available
+
+# TODO: Enable _int_mm in torch
+# if torch.__version__ >= (2, 9):
+#     @register_kernel("bitsandbytes::int8_linear_matmul", "xpu")
+#     def _(A: torch.Tensor, B: torch.Tensor):
+#         return torch._int_mm(
+#             A.reshape(-1, A.shape[-1]),
+#             B.t(),
+#         ).reshape(*A.shape[:-1], B.shape[0])
+
```

Comment:
We can remove these.

Reply:
@jiqing-feng, please help to refine frontend according to @fengyuan14 's comment, thanks!